src/api/app/policies/flag_policy.rb

```diff
@@ -0,0 +1,7 @@
+class FlagPolicy < ApplicationPolicy
+  # just admin are able to create flag
```

Comment:
this is false and misleading. Regular users can't *add* '*access' flags to existant projects. Creating them is allowed for everyone.